src/main/java/xbot/common/subsystems/vision/AprilTagVisionSubsystemCamera.java

```diff
@@ -0,0 +1,162 @@
+package xbot.common.subsystems.vision;
+
+import edu.wpi.first.apriltag.AprilTagFieldLayout;
+import edu.wpi.first.math.VecBuilder;
+import edu.wpi.first.math.geometry.Pose3d;
+import edu.wpi.first.wpilibj.Alert;
+import org.littletonrobotics.junction.Logger;
+import xbot.common.advantage.DataFrameRefreshable;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.PropertyFactory;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class AprilTagVisionSubsystemCamera implements DataFrameRefreshable {
```

Comment:
⭐ wondering if this needs to have Subsystem in the name?

Reply:
Agreed - if it doesn't extend BaseSubsystem, probably shouldn't have Subsystem in the name

Reply:
Renamed